CONTRIBUTING.template.md

```diff
@@ -0,0 +1,55 @@
+# [PROJECT] Contribution and Governance Policies
```

Comment:
Could you please use `{project name}` instead of `[PROJECT]` ?

Reply:
Done 👍